src/App.js

```diff
@@ -1,24 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import ComponentOne from './components/ComponentOne'
```

Comment:
@shivam-innotech change the component name to a better name, ComponentOne is not explaining what it is for.